source/transifex-master/en.json

```
@@ -49,6 +49,8 @@
   "self-host-or-use": "Self-host or use a shared server",
   "use-a-shared": "Use a shared server if you're not tech savvy, or deploy your  own instance on cloud servers or hardware devices.",
   "by-sharing-your": "By sharing your server you can support friends or your local community.",
+  "automation-via-api": "Automation via the API",
+  "automation-via-api-text": "Our API offers programmatic access to your instance. Manage your BTCPay Server (e.g. stores, invoices, users) and automate your workflows.",
```

```suggestion
  "automation-via-api-text": "Manage your BTCPay Server (e.g. stores, invoices, users), automate your workflows and control access with our API.",
 ```
 
  Wording change. `Programmatic` is highly likely to be impossible to translate in a number of languages.

---

Thank you, updated! 👍